OpenRA.Mods.D2k/Graphics/SonicBlastRenderable.cs

```diff
@@ -0,0 +1,66 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using OpenRA.Graphics;
+using OpenRA.Mods.D2k.Traits;
+using OpenRA.Primitives;
+
+namespace OpenRA.Mods.D2k.Graphics
+{
+	public class SonicBlastRenderable : IRenderable, IFinalizedRenderable
+	{
+		public static readonly IEnumerable<IRenderable> None = Array.Empty<IRenderable>();
+		readonly SonicBlastRenderer renderer;
+		readonly float3 r;
+
+		public WPos Pos { get; }
+		public int ZOffset { get; }
```

Comment:
```suggestion
		public int ZOffset => 0;
```

ZOffset is unused

Reply:
It is assigned the value passed from the projectile, and then used by WorldRenderer…?

Reply:
This is a post process effect. We don't respect z order